app/renderer/js/pages/preference/general-section.ts

```diff
@@ -92,6 +92,34 @@ export default class GeneralSection extends BaseSection {
 				</div>
 				<div class="title">${t.__('Advanced')}</div>
 				<div class="settings-card">
+
+					<div class="setting-row" id="app-language">
+						<div class="setting-description">${t.__('App Laguage (requires refresh)')}</div>
```

Comment:
@akashnimare We should go or `requires restart` here? Since the menu bar needs to be rebuild.

Reply:
I was also ambiguous so kept refresh for the time being, now changed to restart
If you/ @akashnimare  think otherwise, I will revert it

Reply:
Yeah, gotta add restart here.